.gitignore

```diff
@@ -33,5 +33,5 @@ htmlcov/
 # test results
 /api/tests/results.xml
 
-/docker/var/minio-data
+/**/var/minio-data
```

Comment:
Do you have a separate dev process that put files in `{repo}/var` still? I want to do what I can to support all people's dev environments, without locking us into one particular way, so keen to hear about things that are painful for you.

Reply:
I think it was old docker instance... so this change isn't really necessary